components/zcash/rs/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "zcash-cxx"
+version = "1.0.0"
+edition = "2021"
+license = "MPL-2.0"
+
+[dependencies]
+orchard = { version = "0.7.1",  default-features = false }
+zip32 = { version = "0.1.1" }
```

Comment:
If you only access zip32 through orchard, you don't need to declare it separately.